apps/system-e2e/src/support/commands.ts

```diff
@@ -60,18 +60,18 @@ Cypress.Commands.add('patchSameSiteCookie', (interceptUrl) => {
   }).as('sameSitePatch')
 })
 
-Cypress.Commands.add('idsLogin', ({ phoneNumber }) => {
+Cypress.Commands.add('idsLogin', ({ phoneNumber, url }) => {
   cy.log('foobar', testEnvironment)
   if (testEnvironment !== 'local') {
     cy.session('idsLogin', () => {
       cy.session('cognitoLogin', () =>
         cognitoLogin({ cognitoUsername, cognitoPassword }),
       )
-      idsLogin(phoneNumber)
+      idsLogin(phoneNumber, url)
     })
   } else {
     cy.session('idsLogin', () => {
-      idsLogin(phoneNumber)
+      idsLogin(phoneNumber, url)
     })
   }
```

Comment:
Could be simplified to have `cy.session` wrapping the `if`-statement

Reply:
That might be a different refactoring ;). This one was getting rid of the hardcode path, so I think it is better to stick to that purpose.